### DISCUSSION TITLE
setting slots for a mod pack

### DISCUSSION THREAD

**Discussion by deadlywolf301** [TheIllusiveC4/Curios#220] [Q&A]
so I've been playing around with this mod and I'm trying to set my own slots instead of the pre configured ones and I cant seem to do it any other way than to load the world in creative and set the slots manually via commands if you could help me out in any way that would be super thanks. Ps, I'm not trying to add custom slots just setting the number of pre configured slots to what I want and I want this to work for when I release this pack to the public.

**Comment by TheIllusiveC4** [comment]
Have you taken a look at this wiki page?
https://github.com/TheIllusiveC4/Curios/wiki/How-to-Use:-Users

It describes how you can set up or configure existing slots using the `curios-server.toml` file. If you use the identifier of a slot type that already exists, you'll change the existing slots instead of making a new type.

**Reply by Pybrid** [reply]
Im in the same situation as dadlywolf301. I've made a modpack that generated the slots, but I want to modify them by setting a custom amount of each slot, as I was given 4 belt slots but only 1 charm slot and wasn't given slots for Arcane Scroll or Spellstone that Engimatic Legacy needs. I used the wiki, but came out with still having 4 belt slots plus 2 new ones without the icon and named curios.identifier.belt

**Reply by Pybrid** [reply]
Just tested, it also doesn't work as a slot to put anything in.

**Reply by Pybrid** [reply]
New information: I've learned that the spellstone and scroll slots are "locked" and only become visible when gaining an item to fill it. The other situation still applies however.
